runtime/default.policy

```diff
@@ -1,82 +1,133 @@
 ---
-version: 1.0.2
+version: 1.1.0
+
+#################################################################
+# Macros are a way to define reusable expressions and/or lists. #
+# Macros are used within rules. Macros do not monitor alone.    #
+#################################################################
+macros:
+  - id: DD_AGENT_PROCESSES
+    description: Processes that are a part of the Datadog Agent
+    expression: >-
+      ["/opt/datadog-agent/embedded/bin/agent", "/opt/datadog-agent/embedded/bin/system-probe",
```

Comment:
There is an issue with the SECL lexer, we can't have a new line in the middle of a list. It generates a syntax error. We are going to address that but meanwhile we need to have the list in a single line

Reply:
Good to know, I will make these single line for now, and we can re-format when the lexer can handle it